src/main/java/mate/academy/hibernate/relations/service/ActorServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.hibernate.relations.service;
+
+import mate.academy.hibernate.relations.DataProcessingExeption;
```

Comment:
There is a typo in the imported exception class `DataProcessingExeption`. It should be `DataProcessingException` to match the corrected class name.